src/main/java/sirius/web/data/RowProcessor.java

```diff
@@ -14,15 +14,26 @@
  * Invoked by a {@link LineBasedProcessor} to handle one row.
  */
 public interface RowProcessor {
+
     /**
      * Called the handle a row of an input file.
      * <p>
-     * Note that an exception thrown here will completely abort the whole process. Therefore proper exception
-     * handling is required.
+     * Note that an exception thrown here will completely abort the whole process if {@link #handleError(Exception)} is
+     * not implemented. Therefore proper exception handling is required.
      * </p>
      *
      * @param lineNumber the line number which is currently being processed.
      * @param row        the data of the row / line to process
      */
     void handleRow(int lineNumber, Values row);
+
+    /**
+     * Called if an {@link Exception} is thrown while parsing a row.
+     *
+     * @param error The {@link Exception} which was thrown
+     * @return <tt>true</tt> to continue execution, <tt>false</tt> to abort execution
+     */
+    default boolean handleError(Exception error) {
```

Comment:
why not require this to be present?

Reply:
the thinking was, like that it still is a functional interface and can be used as lambda. also so old code does not have to change and we don't have to implement a handleError-method everywhere

But I am not sure the method really belongs in this interface, maybe a "withErrorHandler" in "LineBasedProcessor" would be smarter? what do you think?